wagtail/wagtailadmin/templates/wagtailadmin/shared/breadcrumb.html

```diff
@@ -1,8 +1,10 @@
 {% load i18n wagtailadmin_tags %}
 
 <ul class="breadcrumb">
-    {% for page in page.get_ancestors %}
-        {% if page.is_root %}
+    {% explorable_ancestors page request.user as ancestors %}
+    {% for page in ancestors %}
+        {% is_explorable_root page request.user as page_is_explorable_root %}
+        {% if page_is_explorable_root %}
```

Comment:
That sounds like a bug, since "Blog Post 3" should be visible as text, with "Blog Index" being linked as that home icon. I'll look into it.